test/jdk/java/awt/print/Dialog/DialogType.java

```diff
@@ -26,49 +26,58 @@
  * @bug 6568874
  * @key printer
  * @summary Verify the native dialog works with attribute sets.
- * @run main/manual=yesno DialogType
+ * @library /java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
+ * @run main/manual DialogType
  */
 
-import java.awt.print.*;
-import javax.print.attribute.*;
-import javax.print.attribute.standard.*;
+import java.awt.print.PrinterJob;
 
-public class DialogType {
+import javax.print.attribute.Attribute;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.DialogTypeSelection;
+
+import jtreg.SkippedException;
 
-  static String[] instructions = {
-     "This test assumes and requires that you have a printer installed",
-     "It verifies that the dialogs behave properly when using new API",
-     "to optionally select a native dialog where one is present.",
-     "Two dialogs are shown in succession.",
-     "The test passes as long as no exceptions are thrown, *AND*",
-     "if running on Windows only, the first dialog is a native windows",
-     "control which differs in appearance from the second dialog",
-     ""
-  };
+public class DialogType {
+    private static PrinterJob job;
 
-  public static void main(String[] args) {
+    private static final String INSTRUCTIONS = """
+        This test assumes and requires that you have a printer installed.
+        It verifies that the dialogs behave properly when using new API
+        to optionally select a native dialog where one is present.
+        Two dialogs are shown in succession.
+        The test passes as long as no exceptions are thrown.
+        Note: On Mac OS & on Windows, the first dialog is a native
```

Comment:
May add a blank line before Note.
```suggestion
        The test passes as long as no exceptions are thrown.
        
        Note: On Mac OS & on Windows, the first dialog is a native
```